docs/articles/documentation/test-api/selecting-page-elements/selectors/using-selectors.md

```diff
@@ -181,9 +182,20 @@ method if you use API or specify the [selector-timeout](../../../using-testcafe/
 if you run TestCafe from the command line.
 
 Within the selector timeout, the selector is executed over and over again, until it returns a
-DOM node or the timeout exceeds. If TestCafe cannot find the corresponding node in the DOM, the test fails. The test report shows a sequence of the selector's methods that were called during the test and highlights the failed method. This helps you understand what caused the fail.
-
-![Selector methods in reports](../../../../images/failed-selector-report.png)
+DOM node or the timeout exceeds. If TestCafe cannot find the corresponding node in the DOM, the test fails.
 
 > Note that you can require that the node returned by the selector is visible.
-To do this, use the [visibilityCheck](selector-options.md#optionsvisibilitycheck) option.
\ No newline at end of file
+To do this, use the [visibilityCheck](selector-options.md#optionsvisibilitycheck) option.
+
+## Debug Selectors
```

Comment:
Should we add a link to this topic in [FAQ](https://devexpress.github.io/testcafe/faq/#my-test-fails-because-testcafe-could-not-find-the-required-webpage-element-why-does-this-happen)?

Reply:
Like... Make it the first step in this section saying 

```
Look at the report's error message to learn [which selector has failed](#debug-selectors). Add the t.debug() method before this selector to stop test execution before it reaches this selector... bla-bla-bla
```

?